ui/ui.js

```diff
@@ -171,6 +171,7 @@ shaka.ui.Overlay = class {
   defaultConfig_() {
     const config = {
       controlPanelElements: [
+        'close',
```

Comment:
I'm not sure if it should be on by default.  If not, the demo app would enable it explicitly in its own config.  Anyone else have opinions about this?

Reply:
Actually I also had the doubt for if to make it on by default. I can make changes as everyone suggests? @ismena @michellezhuogg

Reply:
Let's turn it off by default, and enable it in the demo app's UI config.  Please also remove the demo-specific implementation in demo/close_button.js.  Thanks!